_includes/setup/system-linux.md

```diff
@@ -0,0 +1,8 @@
+## System requirements
```

Comment:
When navigating to either of setup_linux, setup_mac, or setup_win, the whole sidenav collapses:

![screen shot 2017-03-24 at 13 06 34](https://cloud.githubusercontent.com/assets/13644170/24293569/bc1f7f5a-1092-11e7-8dd4-53f5b73517d7.png)